### PR TITLE
Add description for vTaskList

### PR DESCRIPTION
**Pull request by gomonovych**
Description
-----------

Describe each column which `vTaskList` print:
```
task name, task status, task priority, task stack unused watermark level, task number
```
Add description in function comment about columns which vTaskList print. Maybe like a next step it will be useful to add one top row with column descriptions?

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by n9wxu — APPROVED** [review]

File: include/task.h [hunk 1]
```diff
@@ -1795,8 +1795,10 @@ UBaseType_t uxTaskGetSystemState( TaskStatus_t * const pxTaskStatusArray,
  * demo applications.  Do not consider it to be part of the scheduler.
  *
  * vTaskList() calls uxTaskGetSystemState(), then formats part of the
- * uxTaskGetSystemState() output into a human readable table that displays task
- * names, states and stack usage.
+ * uxTaskGetSystemState() output into a human readable table that displays task:
+ * names, states, priority, stack usage and task number.
+ * Stack usage specified as the number of unused variables stack can hold
```
Comment (RichardBarry) [inline include/task.h:1800]: This is very imprecise as variables can be almost any number of bytes.  Does it mean words?  A word being 16-bits on 16-bit devices, 32-bits on 32-bit devices?

File: tasks.c [hunk 2]
```diff
@@ -4434,7 +4434,9 @@ static void prvResetNextTaskUnblockTime( void )
          *
          * vTaskList() calls uxTaskGetSystemState(), then formats part of the
          * uxTaskGetSystemState() output into a human readable table that
-         * displays task names, states and stack usage.
+         * displays task: names, states, priority, stack usage and task number.
```
Comment (RichardBarry) [inline tasks.c:4437]: Same comment as per prevision.

**Review by RichardBarry — CHANGES_REQUESTED** [review]

File: tasks.c (hunk 2 above)
Comment (gomonovych) [reply tasks.c:4437]: Replaced with: Stack usage specified as the number of unused StackType_t words

File: include/task.h (hunk 1 above)
Comment (gomonovych) [reply include/task.h]: Replaced with: Stack usage specified as the number of unused StackType_t words

**Review by n9wxu — APPROVED** [review]

**Pull request merged**